Scenarios/ODataUniversalApp/ODataUniversalApp/Package.appxmanifest

```diff
@@ -1,48 +1,27 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-
```

Comment:
Please leave comments in the commit about why we should update this.

Reply:
The certificate expired. The build was failing without updating the certificate.